src/app/app.component.html

```diff
@@ -34,6 +52,13 @@
       </button>
     </div>
 
+    <div fxShow="true" fxHide.gt-sm="true">
+      <button mat-icon-button (click)="sidenav.toggle()">
+        <mat-icon>Services</mat-icon>
```

Comment:
Is this intended? It looks like this to me

![image](https://user-images.githubusercontent.com/26195439/159192856-13e6fcb0-32bf-4484-88e3-80df014136db.png)

Reply:
Fixed